src/components/BottomBar.tsx

```diff
@@ -0,0 +1,175 @@
+'use client';
+
+import { Box, Typography, Button, Menu, MenuItem } from '@mui/material';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useCallback, useEffect, useState } from 'react';
+
+export default function BottomBar() {
+  const [isOpen, setOpen] = useState<null | HTMLElement>(null);
+  const [currentLanguage, setCurrentLanguage] = useState<string>('Português');
```

Comment:
In the future it might be better to leave the language state in a context (depending on logic), but for now it's fine to leave it here.